tests/build/build_templates.go

```diff
@@ -203,27 +202,49 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 					pipelineRun, err = kubeadminClient.HasController.GetComponentPipelineRun(componentNames[i], applicationName, testNamespace, "")
 					Expect(err).ShouldNot(HaveOccurred())
 				})
+
 				It("should have Pipeline Records", func() {
-					// Verify Records
 					records, err := resultClient.GetRecords(testNamespace, string(pipelineRun.GetUID()))
+					// temporary logs due to RHTAPBUGS-213
+					GinkgoWriter.Printf("records for PipelineRun %s:\n%s\n", pipelineRun.Name, records)
+					GinkgoWriter.Printf("got error getting records: %v\n", err)
 					Expect(err).NotTo(HaveOccurred())
 					Expect(len(records.Record)).NotTo(BeZero(), "No records found for PipelineRun %s", pipelineRun.Name)
 				})
 
 				It("should have Pipeline Logs", func() {
 					// Verify if result is stored in Database
+					// temporary logs due to RHTAPBUGS-213
 					logs, err := resultClient.GetLogs(testNamespace, string(pipelineRun.GetUID()))
+					GinkgoWriter.Printf("logs for PipelineRun %s:\n%s\n", pipelineRun.Name, logs)
+					GinkgoWriter.Printf("got error getting logs: %v\n", err)
 					Expect(err).NotTo(HaveOccurred())
-					Expect(len(logs.Record)).NotTo(BeZero(), "No logs found for PipelineRun %s", pipelineRun.Name)
+
+					timeout := time.Minute * 2
+					interval := time.Second * 10
+					// temporary timeout  due to RHTAPBUGS-213
+					Eventually(func() bool {
+						// temporary logs due to RHTAPBUGS-213
+						logs, err = resultClient.GetLogs(testNamespace, string(pipelineRun.GetUID()))
+						GinkgoWriter.Printf("logs for PipelineRun %s:\n%s\n", pipelineRun.Name, logs)
+						GinkgoWriter.Printf("got error getting logs: %v\n", err)
+						Expect(err).NotTo(HaveOccurred())
+
+						return len(logs.Record) != 0
+					}, timeout, interval).Should(BeTrue(), fmt.Sprintf("timed out when getting logs for PipelineRun %s", pipelineRun.Name))
```

Comment:
this way the error gets propagated to the error log

more info about this approach [here](https://github.com/redhat-appstudio/e2e-tests/pull/523#discussion_r1198513143) and [here](https://github.com/redhat-appstudio/e2e-tests/pull/531#discussion_r1198641726)
```suggestion
					Eventually(func() (bool, error) {
						logs, err = resultClient.GetLogs(testNamespace, string(pipelineRun.GetUID()))
						// temporary logs due to RHTAPBUGS-213
						GinkgoWriter.Printf("logs for PipelineRun %s:\n%s\n", pipelineRun.Name, logs)

						return len(logs.Record) != 0, err
					}, timeout, interval).Should(BeTrue(), fmt.Sprintf("timed out when getting logs for PipelineRun %s", pipelineRun.Name))
```